cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxAudioFocusManager.java

```diff
@@ -36,6 +36,7 @@
     private final static int AUDIOFOCUS_LOST = 1;
     private final static int AUDIOFOCUS_LOST_TRANSIENT = 2;
     private final static int AUDIOFOCUS_LOST_TRANSIENT_CAN_DUCK = 3;
+    public static boolean gainAudioFocus = true;
```

Comment:
Please change default to false.
@mars3142 Could you support me :)

Reply:
ok but default cocos behaviour was false

Reply:
For me that was regression introduced in 3.14 https://github.com/cocos2d/cocos2d-x/issues/18465
I believe that be default user should be able to play music while playing

Reply:
@newnon Could you apply requested changes? 
My users complains all the time about that they can listen music while playing ...

Reply:
@newnon could you push this PR to master

Reply:
> ok but default cocos behaviour was false

@newnon , the default behavior is true as I know. 
@crazyhappygame , if we change it to false, it will break the compatibility. :(  Any thoughts?

By the way, the member variable naming in Java code should follow `m` as prefix.